Sources/Controllers/Cells/OABaseCell.m

```diff
@@ -0,0 +1,18 @@
+//
+//  OABaseCell.m
+//  OsmAnd Maps
+//
+//  Created by nnngrach on 06.05.2021.
+//  Copyright © 2021 OsmAnd. All rights reserved.
+//
+
+#import "OABaseCell.h"
+
+@implementation OABaseCell
+
++ (NSString *) getCellIdentifier
+{
+    @throw [NSException exceptionWithName:@"OABaseCell error" reason:@"Cell identifier is not defined in cell class" userInfo:nil];
```

Comment:
Instead of thrrowing an exception, I think you should return ``` NSStringFromClass(self.class)```. Please test it and if it works, delete this method implementation in subclasses. However, you need to ensure that each cell identifier is identical to the cell class name.